documentation/resources/03_projects.md

```diff
@@ -0,0 +1,14 @@
+---
+id: projects
+title: Built on Leo
+sidebar_label: Built on Leo
+---
+
+<!--TODO:-->
+
+Coming soon!
+
+:::tip
+Please fill out [this](https://forms.gle/6bt7a65hFBV9MpqN6) form if you're building on Leo.
```

Comment:
The link has a green underscore, which is not visible on the green background. Perhaps the simplest solution is to get rid of the tip wrapper.